Source/WarpX.H

```diff
@@ -256,17 +256,28 @@ public:
     //! Order of finite centering of currents (from nodal grid to staggered grid), along z
     static int current_centering_noz;
 
-    // Number of modes for the RZ multimode version
+    //! Number of modes for the RZ multi-mode version
     static int n_rz_azimuthal_modes;
+    //! Number of MultiFab components
+    //! (with the RZ multi-mode version, each mode has a real and imaginary component,
+    //! except mode 0 which is purely real: component 0 is mode 0, odd components are
+    //! the real parts, even components are the imaginary parts)
     static int ncomps;
 
+    //! If true, a Numerical Cherenkov Instability (NCI) corrector is applied
+    //! (for simulations using the FDTD Maxwell solver)
     static bool use_fdtd_nci_corr;
     static bool galerkin_interpolation;
```

Comment:
Taken from openPMD-standard:
```suggestion
    //! If true (Galerkin method): The fields are interpolated directly from the staggered
    //! grid to the particle positions (with reduced interpolation order in the parallel
    //! direction). This scheme is energy conserving in the limit of infinitely small time
    //! steps.
    //! Otherwise, "momentum conserving" (in the same limit): The fields are first
    //! interpolated from the staggered grid points to the corners of each cell, and then
    //! from the cell corners to the particle position (with the same order of interpolation
    //! in all directions). This scheme is momentum conserving in the limit of infinitely
    //! small time steps.
    static bool galerkin_interpolation;
```